commcare-cloud-bootstrap/environment/app-processes.yml.j2

```diff
@@ -5,7 +5,7 @@ celery_processes:
     flower: {}
     background_queue,case_rule_queue,celery,analytics_queue,case_import_queue:
       concurrency: 1
-    email_queue,reminder_case_update_queue,export_download_queue,linked_domain_queue,malt_generation_queue:
+    email_queue,reminder_case_update_queue,export_download_queue,linked_domain_queue,malt_generation_queue, soltech_custom_script_queue:
```

Comment:
is this for the monolith setup? if yes, I don't think we need this change here?

Reply:
This seems to be redudant code used to set up a new environment. Agreed it's then not necessary here, will remove.

Reply:
[77c363f](https://github.com/dimagi/commcare-cloud/pull/6335/commits/77c363f8ff6952386812d6eb3fd87a87f1ad7493)